Makefile

```diff
@@ -184,6 +184,9 @@ clean:
 	@rm -rf test/e2e/log
 	@rm -rf e2e.namespace
 
+lint:
+	find . -name '*.go' -not -path "./vendor/*" -not -path "./pkg/lib/operatorclient/operatorclientmocks/*" | xargs gofmt -w
```

Comment:
Note: mentioned this in the commit, but that operatorclientmocks package doesn't have the `//go:generate ...` comment that it looks like gofmt/goimports are configured to ignore, so we'd need to explicitly ignore this path from the find command output.